src/ValueFormatters/QuantityUnitFormatter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace ValueFormatters;
+
+/**
+ * Interface defining a service for formatting a quantity's unit.
+ *
+ * @licence GNU GPL v2+
+ * @author Daniel Kinzler
+ */
+interface QuantityUnitFormatter {
+
+	/**
+	 * Attaches a human readable version of the unit to the given number text.
+	 * The number text is expected to already be formatted, and is not modified.
+	 *
+	 * The unit may be placed before or after the number, with or without a separator.
+	 * The unit may also be omitted in case the $unit represents "no unit".
+	 *
+	 * @param string $unit
+	 * @param string $numberText
+	 *
+	 * @return string A text representing the given number with the given unit applied.
+	 */
+	public function applyUnit( $unit, $numberText );
```

Comment:
It's called "amount" and "unit" in `QuantityValue`. Having a "number text" is a bit confusing. What about `$formattedAmount` or something like that?